pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Prada2013/electrolyte_conductivity_Nyman2008.py

```diff
@@ -0,0 +1,33 @@
+from pybamm import exp, constants
+
+
+def electrolyte_conductivity_Nyman2008(c_e, T):
+    """
+    Conductivity of LiPF6 in EC:EMC (3:7) as a function of ion concentration. The data
+    comes from [1].
+    References
+    ----------
+    .. [1] A. Nyman, M. Behm, and G. Lindbergh, "Electrochemical characterisation and
+    modelling of the mass transport phenomena in LiPF6-EC-EMC electrolyte,"
+    Electrochim. Acta, vol. 53, no. 22, pp. 6356–6365, 2008.
+    Parameters
+    ----------
+    c_e: :class:`pybamm.Symbol`
+        Dimensional electrolyte concentration
+    T: :class:`pybamm.Symbol`
+        Dimensional temperature
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        Solid diffusivity
+    """
+
+    sigma_e = (
+        0.1297 * (c_e / 1000) ** 3 - 2.51 * (c_e / 1000) ** 1.5 + 3.329 * (c_e / 1000)
+    )
+
+    E_k_e = 34700
```

Comment:
Redundant if we remove this instance on Nyman, but in the paper there is no temperature dependence so we should remove activation energies.